plugin-server/src/utils/db/db.ts

```diff
@@ -1362,11 +1399,18 @@ export class DB {
         if (!rawActions.length) {
             return null
         }
-        const steps: ActionStep[] = (
```

Comment:
Note: The type here was previously wrong and this meant that if a action was ever reloaded, it would be incorrectly matched. 

We should be writing more unit tests! :sweat_smile: